src/llm/http_llm_calculator.cc

```diff
@@ -266,7 +266,8 @@ class HttpLLMCalculator : public CalculatorBase {
         } catch (...) {
             return absl::InvalidArgumentError("Response generation failed");
         }
-        timestamp = timestamp.NextAllowedInStream();
+        auto now = std::chrono::system_clock::now();
+        timestamp = ::mediapipe::Timestamp(std::chrono::duration_cast<std::chrono::microseconds>(now.time_since_epoch()).count());
```

Comment:
the name is unfortunate here
```suggestion
        iterationBeginTimestamp = ::mediapipe::Timestamp(std::chrono::duration_cast<std::chrono::microseconds>(now.time_since_epoch()).count());
```